.github/workflows/destroy-frontend-preview.yml

```
@@ -0,0 +1,351 @@
+          echo "🌍 Environment name: ${ENV_NAME}"
+
+      - name: Generate Amplify app info
+        id: amplify-info
+        run: |
+          AMPLIFY_APP_NAME="macro-ai-frontend-pr-${{ github.event.pull_request.number }}"
+          echo "amplify-app-name=${AMPLIFY_APP_NAME}" >> $GITHUB_OUTPUT
+          echo "📱 Amplify app name: ${AMPLIFY_APP_NAME}"
+
+      - name: Make teardown decision
+        id: decision
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:33: Double quote to prevent globbing and word splitting

(shellcheck)